### ISSUE TITLE
Method Reproduction

### ISSUE THREAD

**Issue by yarks165** [sev777/DE-VQA#2]
A very excellent article! I want to replicate the training method in the paper by adding a new KL divergence. May I ask where the current warehouse implements this loss function.
Also, regarding The `vlkec_embeddings_collect.ckl` and `vqa_ embeddings_llava.ckl`, do I need to prepare them before evaluation and how can I obtain them. Could you please provide a detailed explanation.
Thank you!
